### DISCUSSION TITLE
How to code a similar Screen

### DISCUSSION THREAD

**Discussion by Steven2k6** [KlipperScreen/KlipperScreen#1089] [General]
Hey, I'd like to program a code to display a custom text on a csi display of a raspberry pi and I'm really inspired and impressed by your work.
That's why I wanted to ask you how you coded klipperscreen.

**Comment by alfrix** [comment]
KlipperScreen is a gtk based application, you should look at https://pygobject.readthedocs.io/en/latest/ and python gtk tutorials to start

**Reply by Steven2k6** [reply]
> KlipperScreen is a gtk based application, you should look at https://pygobject.readthedocs.io/en/latest/ and python gtk tutorials to start

Thank you.

**Reply by Steven2k6** [reply]
> KlipperScreen is a gtk based application, you should look at https://pygobject.readthedocs.io/en/latest/ and python gtk tutorials to start

How exactly did you test the script?

**Reply by Vasilyev81** [reply]
Hi! I'm testing this application on real hardware (Raspberry Pi also)
Do you know Python?
